### DISCUSSION TITLE
loops samples for 15 sec or more. imposible work fine

### DISCUSSION THREAD

**Discussion by Uhboy** [hydrogen-music/hydrogen#1424] [General]
somebody can tell me how I can use a complete loop sample of 15sec or more sec without cut it.?
Or if is posible use the only imported loop sample as patterns [I know this is posible] but how I can adjust perfectly the tempo or bpm without destroy the loop?.
Ps. I'm newbie, also I read the guide but I don't find this answer.
Greetings

**Comment by theGreatWhiteShark** [comment]
I'm not quite sure I understand your problem. In the Instrument Editor you can add samples (layers) of arbitrary length (I just tried it with a >100MB WAV file).

Could you explain step by step what you are doing and what behaves different from what you expected?
